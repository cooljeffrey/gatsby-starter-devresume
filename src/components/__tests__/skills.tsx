import React from "react";
import renderer from "react-test-renderer";
import Skills from "../skills";
describe("Skills", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Skills
          technicalSkills={["skill 1", "skill 2"]}
          professionalSkills={["pro 1", "pro 2"]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
