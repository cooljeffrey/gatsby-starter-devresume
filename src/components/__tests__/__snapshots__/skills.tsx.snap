// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills renders correctly 1`] = `
<section
  className="skills-section py-3"
>
  <h3
    className="text-uppercase resume-section-heading mb-4"
  >
    Skills
  </h3>
  <div
    className="item"
  >
    <h4
      className="item-title"
    >
      Technical
    </h4>
    <ul
      className="list-unstyled resume-skills-list"
    >
      <li
        className="mb-2"
      >
        skill 1
      </li>
      <li
        className="mb-2"
      >
        skill 2
      </li>
    </ul>
  </div>
  <div
    className="item"
  >
    <h4
      className="item-title"
    >
      Professional
    </h4>
    <ul
      className="list-unstyled resume-skills-list"
    >
      <li
        className="mb-2"
      >
        pro 1
      </li>
      <li
        className="mb-2"
      >
        pro 2
      </li>
    </ul>
  </div>
</section>
`;
